Generator/DataMapper/DataTypeMapper.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace WsSys\DtoGeneratorBundle\Generator\DataMapper;
+
+/**
+ * Mapping of Data Types between DTO/XSD or Other types
+ */
+class DataTypeMapper
+{
+    /**
+     * List of mappings
+     * 
+     * @var array 
+     */
+    protected static $xsdToDtoMapping = array(
+        'xs:string' => 'string',
+        'xs:date' => 'DateTime',
+        'xs:datetime' => 'DateTime',
+        'xs:integer' => 'integer',
+        'xs:decimal' => 'double',
+        'xs:positiveInteger' => 'integer',
+        'xs:boolean' => 'boolean'
+        );
+    
+    /**
+     * Maps Data Type from Xsd to Dto
+     * 
+     * @param string $xsdType
+     * 
+     * @return string
+     */
+    public static function XsdToDto($xsdType)
```

Comment:
It is not a good idea to have this class as static as this will not allow it to be testable as well as introduce problems with tests in classes using it.